r55/src/exec.rs

```diff
@@ -186,7 +186,10 @@ fn execute_riscv(
                         let data_bytes = dram_slice(emu, ret_offset, ret_size)?;
 
                         let total_cost = r55_gas + evm_gas;
-                        println!("evm gas: {}, r55 gas: {}, total cost: {}", evm_gas, r55_gas, total_cost);
+                        println!(
+                            "evm gas: {}, r55 gas: {}, total cost: {}",
+                            evm_gas, r55_gas, total_cost
+                        );
```

Comment:
Just curious; which formatter are you using?

Reply:
good ol `cargo fmt`